### PR TITLE
Fix arrays concept

### PR DESCRIPTION
**Pull request by mk-mxp**
The issue was reported in #661, I added some minor improvements in wording and formatting.

### CODE REVIEW

File: concepts/arrays/about.md [hunk 1]
```diff
@@ -36,17 +36,17 @@ $my_empty_arr = [];
...
 // or
-$letters = explode("Hello", ""); // => ["H", "e", "l", "l", "o"] 
+$letters = mb_str_split("Hello"); // => ["H", "e", "l", "l", "o"] 
 ```
 
 Access, assign, append values using the index operator:
 
 ```php
-$prime_numbers = [1, 3, 5, 7, 11, 12];
+$prime_numbers = [1, 3, 5, 6];
```
Comment (homersimpsons) [inline concepts/arrays/about.md:45]: 1 is not a prime number. 2 is the first prime number.


```suggestion
$prime_numbers = [2, 3, 5, 6];
```

File: concepts/arrays/about.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ $no_keys == $integer_keys // => equal returns true
 $no_keys === $integer_keys // => strictly equal returns true
 ```
 
-A value can be of `mixed` type, and each value in an array is not required to be of the same type.
+All values are of `mixed` type, and each value in an array is not required to be of the same type.
```
Comment (homersimpsons) [inline concepts/arrays/about.md:18]: I do not think we should speak about `mixed` here, and I find the sentence hard to read.

The PHP documentation just says:
> The value can be of any type

I can suggest the following but feel like it is not the best either.
```suggestion
Each value can have a different type. There is no type restriction.
```

File: concepts/arrays/introduction.md [hunk 3]
```diff
@@ -23,9 +23,9 @@ Arrays can be declared as a literal (written in code, as done above) or created
 Access, assign, append values using the index operator:
 
 ```php
-$prime_numbers = [1, 3, 5, 7, 11, 12];
+$prime_numbers = [1, 3, 5, 6];
```
Comment (homersimpsons) [inline concepts/arrays/introduction.md:26]: 1 is not a prime number, 2 is the first prime number


```suggestion
$prime_numbers = [2, 3, 5, 6];
```

File: exercises/concept/language-list/.docs/introduction.md [hunk 4]
```diff
@@ -25,11 +25,11 @@ Arrays can be declared as a literal (written in code, as done above) or created
 Access, assign, append values using the index operator:
 
 ```php
-$prime_numbers = [1, 3, 5, 7, 11, 12];
+$prime_numbers = [1, 3, 5, 6];
```
Comment (homersimpsons) [inline exercises/concept/language-list/.docs/introduction.md:28]: 1 is not a prime number, 2 is the first prime number.


```suggestion
$prime_numbers = [2, 3, 5, 6];
```

**Review by homersimpsons — COMMENTED** [review]
Thanks for the improvements. I added some comments to try to get an even better documentation.

File: concepts/arrays/about.md (hunk 2 above)
Comment (mk-mxp) [reply concepts/arrays/about.md:18]:
```suggestion
An array can have values of any type.
Each value can have a different type.
```

Comment (homersimpsons) [reply concepts/arrays/about.md:18]: > @homersimpsons How about that?

@mk-mxp this one looks fine!

**Review by homersimpsons — APPROVED** [review]

**Pull request merged**